web3/geth.py

```diff
@@ -33,6 +33,31 @@
     sign,
     unlockAccount,
 )
+from web3.shh import (
+    addPrivateKey,
```

Comment:
I think this large number of imports could be sufficient justification to just import the `ssh` module and to access the methods as properties of the module (if you're inclined to do it that way).